client/src/graphql/snapshotsQuery.js

```diff
@@ -43,7 +43,6 @@ query ReadSnapshotsPage ($page_id: ID!, $limit: Int!, $offset: Int!) {
             }
             LiveVersion
             LatestDraftVersion
-            LastEdited
```

Comment:
change looks all good but could we re-word the commit message/issue so its a bit clearer what the change is solving?